python/taichi/lang/matrix.py

```diff
@@ -985,10 +997,9 @@ def fill(self, val):
             >>> A
             [-1, -1, -1, -1]
         """
-        def assign_renamed(x, y):
-            return ops_mod.assign(x, y)
-
-        return self._element_wise_writeback_binary(assign_renamed, val)
+        # pylint: disable=C0415
+        from taichi.lang.matrix_ops import fill
+        return fill(self, val)
```

Comment:
Since we have multiple functions named `fill`, let's explicitly call `matrix_ops.fill()` here